test/functional/sqlsrv/sqlsrv_ae_output_param_sqltype_decimal_precision.phpt

```diff
@@ -0,0 +1,264 @@
+--TEST--
+Test for retrieving encrypted data of decimal types with different precisions and scales as output parameters
+--DESCRIPTION--
+Test implicit conversions between different precisions and scales
+With Always Encrypted, no implicit conversion works for decimal datatypes, the precision and scale specified in the SQLSRV_SQLTYPE must be identical to the precision and scale defined in the column
+Without AlwaysEncrypted, implicit conversion between precisions or scales works if:
+1. From a decimal(m1, m2) column to a SQLSRV_SQLTYPE_DECIMAL(n1, n2) output parameter where n1 - n2 >= m1 - m2
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
+--FILE--
+<?php
+require_once('MsCommon.inc');
+
+$dataTypes = array("decimal", "numeric");
+$precisions = array(/*1 => array(0, 1), 
+                    4 => array(0, 1, 4), 
+                    16 => array(0, 1, 4, 16),*/
+                    19 => array(/*0,*/ 1, 4, 16, 19),
+                    38 => array(/*0,*/ 1, 4, 16, 37/*,38*/));
+$sqlTypes = array("SQLSRV_SQLTYPE_DECIMAL", "SQLSRV_SQLTYPE_NUMERIC");
+$sqltypePrecisions = $precisions;
+$inputValuesInit = array(92233720368547758089223372036854775808, -92233720368547758089223372036854775808);
+$maxInPrecision = 38;
+$directions = array("SQLSRV_PARAM_OUT", "SQLSRV_PARAM_INOUT");
+
+$conn = AE\connect();
+
+foreach($dataTypes as $dataType) {
```

Comment:
should there be a space after foreach? there is in line 34

Reply:
according to the coding standard, there should be a space after foreach :( I'll add a space in for all tests